model.go

```diff
@@ -163,18 +172,33 @@ func (n *Node) Summary() string {
 	return summary
 }
 
+// Summary help string for the node (not including application name).
+// Default behaviour used throughout the application
+func (n *Node) Summary() string {
+	return n.SummaryWithOptions(false)
+}
+
 // FlagSummary for the node.
-func (n *Node) FlagSummary(hide bool) string {
```

Comment:
Sorry for the epically late response. Overall the change looks good, but I can't merge this as it's a breaking API change.

Reply:
> Sorry for the epically late response

All good!  my turn this time, was on a short break :)

> I can't merge this as it's a breaking API change.

`FlagSummary` is a fairly short function - so if you don't mind my duplicating it I could turn this into an additive change, ie.

- Additive changes:
   - add `Node.FlagSummaryWithOptions(bool, bool)`
   - `Node.SummaryWithOptions()` calls `Node.FlagSummaryWithOptions(bool, bool)` instead
- Reverted:
   - `Node.FlagSummary()` returns to its original arity
   - `Node.Summary()` goes back to calling `Node.FlagSummary(bool)`

Reply:
Actually - given the current PR is unacceptable, I'll just make the changes now and see what you think :)

Reply:
Rebased, reworked changes to avoid API modification (and attempting to minimise duplication where possible)

Reply:
Just extra explanation re. bonus change in https://github.com/alecthomas/kong/pull/306/commits/b196f4db5d44dfc7b847b60ec65b3af3263546a4 : appears that my rebase picked up some change which is invalid with the current golangci-lint (see previous execution here: https://github.com/alecthomas/kong/actions/runs/3126674306/jobs/5072442276)

Seem fine now